ResearchKit/Common/ORKQuestionStepView.m

```diff
@@ -53,13 +53,41 @@ - (void)setQuestionStep:(ORKQuestionStep *)step {
     self.headerView.instructionLabel.hidden = ![_questionStep text].length;
     
     self.headerView.captionLabel.useSurveyMode = step.useSurveyMode;
-    self.headerView.captionLabel.text = _questionStep.title;
-    self.headerView.instructionLabel.text = _questionStep.text;
+    self.headerView.captionLabel.attributedText = [self getAttributedQuestionTitle];
+    self.headerView.instructionLabel.attributedText = [self getAttributedQuestionText];
     self.continueSkipContainer.optional = _questionStep.optional;
     
     [self.continueSkipContainer updateContinueAndSkipEnabled];
 }
 
+- (NSAttributedString *)getAttributedQuestionTitle {
+    NSString *openFont = @"<font size='6' style='font-family:HelveticaNeue-Light'>";
+    NSString *closeFont = @"</font>";
+    NSString *customTitle = [NSString stringWithFormat:@"%@%@%@", openFont, self.questionStep.title, closeFont];
+    NSError *error = nil;
+    NSMutableAttributedString *attts = [[NSMutableAttributedString alloc] initWithData:[customTitle dataUsingEncoding:NSUnicodeStringEncoding] options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType} documentAttributes:nil error:&error];
+    NSRange range = NSMakeRange(0, attts.length);
+    NSMutableParagraphStyle *style =  [[NSMutableParagraphStyle alloc] init];
+    style.alignment = NSTextAlignmentCenter;
+    [attts addAttribute:NSParagraphStyleAttributeName value:style range:range];
+    
+    return attts;
+}
+
+- (NSAttributedString *)getAttributedQuestionText {
+    NSString *openFont = @"<font size='4' style='font-family:HelveticaNeue-Light'>";
+    NSString *closeFont = @"</font>";
+    NSString *customTitle = [NSString stringWithFormat:@"%@%@%@", openFont, self.questionStep.text, closeFont];
+    NSError *error = nil;
+    NSMutableAttributedString *attts = [[NSMutableAttributedString alloc] initWithData:[customTitle dataUsingEncoding:NSUnicodeStringEncoding] options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType} documentAttributes:nil error:&error];
+    NSRange range = NSMakeRange(0, attts.length);
+    NSMutableParagraphStyle *style =  [[NSMutableParagraphStyle alloc] init];
+    style.alignment = NSTextAlignmentCenter;
+    [attts addAttribute:NSParagraphStyleAttributeName value:style range:range];
+    
+    return attts;
+}
```

Comment:
I'd probably create one method, that let me pass in the font size instead of having two different ones that have almost exactly the same

Reply:
sure